docs/examples/ingest.ipynb

```diff
@@ -0,0 +1,189 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Ingest\n",
+    "\n",
+    "Ingest all guids from AAPB into Chowda\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sonyci import SonyCi\n",
+    "from os import environ\n",
+    "\n",
+    "environ['DB_URL'] = 'sqlite:///../../chowda.development.sqlite'\n",
+    "from tests.factories import MediaFileFactory\n",
+    "\n",
+    "ci = SonyCi.load_from_toml('../../ci.toml')\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_batch(n):\n",
```

Comment:
if we pull this out of a notebook and into real code, we may want to consider different terminology than `batch` to avoid confusion with the db model